docs/contracts_08_other_evm.md

```diff
@@ -12,12 +12,16 @@ The canonical versions of the Gnosis Safe smart contracts are deployed to the fo
 - Energy Web Chain
 - Energy Web Chain Testnet: Volta
 
-The contract addresses match on all these networks. The full list can be found in the `networks.json` of the respective release in the Gnosis Safe contracts repository on Github (e.g. for [v.1.2.0](https://github.com/gnosis/safe-contracts/blob/v1.2.0/networks.json)).
+The contract addresses match on all these networks. The full list can be found in the [Safe deployments](https://github.com/gnosis/safe-deployments) repository on Github (e.g. for [v.1.3.0](https://github.com/gnosis/safe-deployments/tree/main/src/assets/v1.3.0)).
 
-In case you are missing the canonical Gnosis Safe contracts on another EVM-based chain, instructions on how to set them up can be found [here](https://github.com/gnosis/safe-contract-deployment-replay).
+To deploy the Gnosis Safe contracts on another EVM-based chain (needs to be fully compatible, e.g. support all opcodes used by the Gnosis Safe contracts) follow the instruction in the Gnosis Safe contract repository [readme](https://github.com/gnosis/safe-contracts/blob/v1.3.0/README.md#custom-networks).
+
+In case you are missing the canonical Gnosis Safe contracts for the versions 1.1.1 or 1.2.0 on another EVM-based chain, instructions on how to set them up can be found [here](https://github.com/gnosis/safe-contract-deployment-replay).
 
 In order to run the [Gnosis Safe Web interface](gnosis-safe.io/app/) ([Code](https://github.com/gnosis/safe-react/)), you would need to also run the backend services, in particular the [Safe client gateway](https://github.com/gnosis/safe-client-gateway/) and the [Safe transaction service](https://github.com/gnosis/safe-transaction-service) including a tracing node.
 
-At Gnosis, we do not have the capacity to spin up and maintain full frontend and backend support for other EVM-based networks. All our code is open source. We encourage everyone to [deploy the canonical versions](https://github.com/gnosis/safe-contract-deployment-replay) of the Safe contracts to their respective networks and run the required backend and frontend parts themselves. 
+At Gnosis, we do not have the capacity to spin up and maintain full frontend and backend support for other EVM-based networks. All our code is open source. We encourage everyone to [deploy the canonical versions](https://github.com/gnosis/safe-contracts/blob/v1.3.0/README.md#custom-networks) of the Safe contracts to their respective networks and run the required backend and frontend parts themselves.
```

Comment:
Should we mention planned L2 support?

Reply:
That is something @lukasschor needs to decide

Reply:
Do we even need to have this section? We have the information above on how to deploy yourself, but we do not necessarily encourage everyone to do it, especially the ones where we have plans to add support ourselves anyways in the near future. What's the objective of this section?

Reply:
Lets adjust this in a different PR ;)